src/licensedcode/data/rules/mit_1354.RULE

```diff
@@ -0,0 +1,31 @@
+---
+license_expression: mit
+is_license_notice: yes
+relevance: 100
+ignorable_copyrights:
+  - "Copyright (c) 2016 Jia PengHui"
+ignorable_holders:
+  - "Jia PengHui"
+---
+
+{{MIT License}}
+
+Copyright (c) 2016 Jia PengHui
```

Comment:
No copyrights in rule please ... unless absolutely needed.  This is not needed here